### DISCUSSION TITLE
Automated build without user interaction

### DISCUSSION THREAD

**Discussion by william-c-simpson** [brentvollebregt/auto-py-to-exe#374] [Ideas]
Would it be possible to add something like a --quiet option that would do the build off of a config file alone without involving the web interface at all? I want to run this as part of an automated pipeline.

**Comment by brentvollebregt** [comment]
I would say just use PyInstaller directly - no need to use auto-py-to-exe then. The command to use is at the bottom of the UI.

**Comment by william-c-simpson** [comment]
That's easy enough, thanks!
